src/test/pages/Issue.test.tsx

```diff
@@ -68,13 +89,11 @@ describe('issue form', () => {
   });
 
   it('the security deposit is correctly displayed', async () => {
```

Comment:
All test case names could be improved to preserve the natural readability of the test, now it reads like 'it the security deposit...' which is not correct. 

See code examples here to understand better what I mean: https://stackoverflow.com/a/56072272

Reply:
Thank you. Let me improve it.

Reply:
Done!